### PR TITLE
Feature/modals in activities

### PR DESCRIPTION
**Pull request by nikonikoniko**

### CODE REVIEW

File: src/app/kid_game/components/modal.cljs [hunk 1]
```diff
@@ -38,6 +38,6 @@
        [:div {:class "modal-background"
               :on-click #(close-modal)}]
        [:div {:class "box column is-5 p-5"
-              :style {:z-index 10}} [content]]
+              :style {:z-index 10}} (if content [content] "no content")]
```
Comment (nikonikoniko) [inline src/app/kid_game/components/modal.cljs:41]: note, would cause a crash when content not provided, because `[nil]` crashes reagent

Comment (basilesimon) [reply src/app/kid_game/components/modal.cljs:41]: Thank you

**Comment by nikonikoniko** [comment]
@basilesimon what do we want to do about this modal - should it always take full screen?

**Comment by basilesimon** [comment]
If you place it within the hub, does it stay "contained" inside there too?

**Comment by nikonikoniko** [comment]
what do you mean? that it is `activity` which constrains it as opposed to `hub`?

**Pull request merged**